src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.cs

```diff
@@ -58,7 +59,7 @@ internal abstract partial class AbstractAsynchronousTaggerProvider<TTag> where T
         /// created for a previous version of a document that are mapped forward by the async
         /// tagging architecture.  This value cannot be <see cref="SpanTrackingMode.Custom"/>.
         /// </summary>
-        protected virtual SpanTrackingMode SpanTrackingMode => SpanTrackingMode.EdgeExclusive;
+        public virtual SpanTrackingMode SpanTrackingMode => SpanTrackingMode.EdgeExclusive;
```

Comment:
This is a breaking change for TS
http://index/?query=Microsoft.CodeAnalysis.Editor.TypeScript.Features.Classifier.SemanticClassificationTaggerProvider&rightProject=Microsoft.CodeAnalysis.TypeScript.EditorFeatures&file=Features%5CClassifier%5CSemanticClassificationTaggerProvider.cs&line=43

Reply:
fixed in followup.